games/devtest/mods/benchmarks/init.lua

```diff
@@ -114,4 +120,29 @@ minetest.register_chatcommand("bench_bulk_set_node", {
 	end,
 })
 
+minetest.register_chatcommand("bench_bulk_get_node", {
+	params = "",
+	description = "Benchmark: Bulk-get 99×99×99 nodes",
+	func = function(name, param)
+		local player = minetest.get_player_by_name(name)
+		if not player then
+			return false, "No player."
+		end
+		local pos_list = get_positions_cube(player:get_pos())
+
+		minetest.chat_send_player(name, "Benchmarking minetest.bulk_get_node...")
 
+		local start_time = minetest.get_us_time()
+		for i=1,#pos_list do
+			minetest.get_node(pos_list[i])
+		end
+		local middle_time = minetest.get_us_time()
+		minetest.bulk_get_node(pos_list, {name = "mapgen_stone"})
+		local end_time = minetest.get_us_time()
+		local msg = string.format("Benchmark results: minetest.get_node loop: %.2f ms; minetest.bulk_get_node: %.2f ms",
```

Comment:
Like set_bulk_node, this is slightly biased because `get_positions_cube` is not taken into account.